server/app/controllers/milestoneController.js

```diff
@@ -0,0 +1,62 @@
+const { sequelize, User, Cohort, Milestone } = require("../../models");
+
+async function HandleCurrentMilestone(req, res) {
+  try {
+    // console.log(req.user)
+    // const email = req.user.dataValues.email;
```

Comment:
Why is all the logic commented out?

Reply:
Because there is no milestone in the current. I just wanted to send something to the frontend for doing next tasks.
we should create some data or fake data and then i will come back and uncomment the codes. what do you think?

Reply:
Let's create a seed file with a fake cohort and milestones.

Reply:
It's a bad practice to add commented-out code to the codebase.